src/sdam/srv_polling.ts

```diff
@@ -23,7 +23,7 @@ class SrvPollingEvent {
     this.srvRecords = srvRecords;
   }
 
-  addresses() {
+  addresses(): Set<string> {
     return new Set(this.srvRecords.map((record: any) => `${record.name}:${record.port}`));
   }
 }
```

Comment:
I can't comment on it, but SrvPoller below looks untyped, is there a reason to leave it off?

Reply:
oops, missed this file it looks like. Updated